angular-with-directus/how to integrate directus with Angular.md

```diff
@@ -0,0 +1,396 @@
+Angular is a front-end framework that leverage TypeScript to create web applications. In this tutorial, you will use the framework to implement the front-end for the Directus headless CMS. You will implement a blog that loads blog posts dynamically and also serves global metadata.
+
+## Prerequisites
+- Some knowledge of TypeScript and [Angular](https://angular.io/)
+- A Directus project either locally or on [Directus cloud](https://directus.cloud/register)
+- [Node.js](https://nodejs.org/en/download) and a development environment of your choice
+- Install the Angular CLI - use the Angular [guide](https://angular.io/guide/setup-local) to achieve this. Note that Angular and TypeScript versions must be compatible. Since the SDK requires a minimum TypeScript version of 5.0, you need to use Angular version 17 for your project.
+
+## Initialize Project
+To create a new Angular project, use the following command.
+
+```bash
+ng new directus-with-angular
+```
+For the prompts, choose CSS for the stylesheet format and no to disable server side rendering and static site generation.
+
+```bash
+? Which stylesheet format would you like to use? CSS
+? Do you want to enable Server-Side Rendering (SSR) and Static Site Generation (SSG/Prerendering)? (y/N) No
+```
+Once the project has been created, open it in the development environment and replace the code in the `src/app/app.component.html` file with the following tag.
+
+```html
+<router-outlet/>
+```
+Angular will dynamically fill the [RouterOutlet](https://angular.io/api/router/RouterOutlet) placeholder based on the current router state.
+
+You should also disable strict checking in `./tsconfig.json` file under `compilerOptions`.
+
+```json
+"strict": false
+```
+
+Next, open a new terminal window and CD into the project then use the following command to start the server at `http://localhost:4200`.
+
+```bash
+ng serve
+```
+## Configure the Directus SDK
+Open a new terminal window and CD into the project then use the following command to install the directus SDK.
+
+```bash
+npm install @directus/sdk
+```
```

Comment:
Move this to the Initialize project step before starting the dev server, then everything can happen in one terminal. 

I also prefer `Navigate to your project directory in a terminal and...`.